db/seeds.rb

```diff
@@ -22,6 +22,12 @@
              activated: true,
              activated_at: Time.zone.now)
 
+users = User.order(:created_at).take(6)
+30.times do
+ content = Faker::Lorem.sentence(word_count: 5)
```

Comment:
you can try replace `faker` to `ffaker` instead
refs: https://github.com/ffaker/ffaker
- Add to Gemfile:
```
group :development, :test do
  # some gems
  gem 'ffaker'
end
```
- At here: 
```suggestion
 content = FFaker::Lorem.sentence(word_count: 5)
```